### ISSUE TITLE
MHCC CTR Not Displaying on Vatspy

### ISSUE THREAD

**Issue by manigoes** [vatsimnetwork/vatspy-data-project#643]
MHTG_CTR's (CENTRAL AMERICA FIR) callsign was renamed to MHCC_CTR in 2021.  MHCC Center displays properly in Simaware but not in Vatspy.  Vatspy shows MHCC_CTR as an Observer callsign even when I log into the CTR facility. See supporting screenshots which are attached.
[MHCC_CTR Display Issue 011223.docx](https://github.com/vatsimnetwork/vatspy-data-project/files/10412909/MHCC_CTR.Display.Issue.011223.docx)

**Comment by manigoes** [comment]
I am attaching a file with coordinates to MHCC_CTR boundaries.
[MHCC_CTR Coordinates.txt](https://github.com/vatsimnetwork/vatspy-data-project/files/10448375/MHCC_CTR.Coordinates.txt)

**Comment by Adrian2k** [comment]
This was fixed 2 years ago in #83 where we renamed it from MHTG to MHCC.

> Vatspy shows MHCC_CTR as an Observer callsign even when I log into the CTR facility. See supporting screenshots which are attached.

The only reason VATSpy will show a controller as an observer is when the controller has not marked any frequency as primary. This has nothing to do with the database set, even if the FIR was missing in our dataset it would not mark it as an observer.

So if the primed frequency is missing (or showing 199.998) it will be marked as an observer until a frequency is primed.


Example from ENBD:
Here I have not activated any frequency, hence it is showing me as an observer.
![image](https://user-images.githubusercontent.com/35482228/216125192-5b96b963-5863-4719-b39b-90a80ceec3bd.png)
![image](https://user-images.githubusercontent.com/35482228/216125343-04d1cd10-cff2-467c-96a5-5746ccb41ef3.png)

But once I set an active frequency, I will be displayed:
![image](https://user-images.githubusercontent.com/35482228/216125528-c842f946-335d-4306-8090-45124308e9bb.png)
